site/_docs/reference.md

```diff
@@ -2832,7 +2832,8 @@ In the following:
 | b | CODE_POINTS_TO_BYTES(integers)                 | Converts *integers*, an array of integers between 0 and 255 inclusive, into bytes; throws error if any element is out of range
 | b | CODE_POINTS_TO_STRING(integers)                | Converts *integers*, an array of integers between 0 and 0xD7FF or between 0xE000 and 0x10FFFF inclusive, into string; throws error if any element is out of range
 | o r | CONCAT(string, string)                       | Concatenates two strings, returns null only when both string arguments are null, otherwise treats null as empty string
-| b m | CONCAT(string [, string ]*)                  | Concatenates one or more strings, returns null if any of the arguments is null
+| b | CONCAT(string [, string ]*)                    | Concatenates one or more strings, returns null if any of the arguments is null
+| m | CONCAT(any [, any ]*)                          | Concatenates one or more strings/bytestrings/..., if arguments contains bytestring would return bytes string and returns null if any of the arguments is null
```

Comment:
Except string and bytestring, what else is supported? If not, maybe you should remove ... .
How about "returns null if any of the arguments is null, and returns bytes string if any of the arguments is a bytestring"

Reply:
As description above, MySQL concat function not only support string/bytestring, also support numberic types(which can cast to string type would all support). so IMO ```...``` maybe necessary.

Reply:
That's not correct. The function only accepts CHAR and BINARY arguments; the other arguments are accepted because the compiler ill insert implicit casts. That's a completely separate issue, which has no bearing to this function.